src/client/pages/Checkout/CheckoutDetails/components/QuoteDetails/QuoteDetails.tsx

```diff
@@ -56,26 +54,13 @@ const getValueText = ({
   return value
 }
 
-export const QuoteDetails = () => {
+export const QuoteDetails = (quoteData: any) => {
   const textKeys = useTextKeys()
-  const { quoteCartId } = useQuoteCartIdFromUrl()
-  const { data } = useQuoteDetailsDataQuery({
-    variables: { id: quoteCartId },
-  })
-
-  const quoteDetailsData = data?.quoteCart.bundle?.quotes?.map(
-    ({ data }) => data,
-  )
-
-  if (!quoteDetailsData) {
-    return null // TODO: Do something more sophisticated
-  }
-
-  const quoteDetails = getQuoteDetails({ quoteDetailsData })
+  const quoteDetails: any[] = quoteData.quoteData
```

Comment:
Let's try to avoid `any` when we are writing new features/code, right?

Reply:
yeah, still working on that one.